Podfile

```diff
@@ -20,18 +20,18 @@ target 'WooCommerce' do
 
   # Use the latest bugfix for coretelephony
   #pod 'Automattic-Tracks-iOS', :git => 'https://github.com/Automattic/Automattic-Tracks-iOS.git', :tag => '0.2.4-beta.1'
-  pod 'Automattic-Tracks-iOS', '~> 0.4.4-beta'
+  pod 'Automattic-Tracks-iOS', '~> 0.4.4-beta.2'
```

Comment:
> TIL that Cocoapods does not like our beta versions and will ignore newer beta versions, so I made the beta versions explicit in the podfile.

That's weird. I changed this back to:

```ruby
  pod 'Automattic-Tracks-iOS', '~> 0.4.4-beta'
```

And then executed: 

```shell
$ rm -rf Pods Podfile.lock
$ bundle exec pod install
```

... and Cocoapods **successfully** fetched the `-beta.2`:

```
Analyzing dependencies
Downloading dependencies
Installing 1PasswordExtension (1.8.6)
Installing Alamofire (4.8.0)
Installing Automattic-Tracks-iOS (0.4.4-beta.2) <----
```

The generated `Podfile.lock` is also pointing to `-beta.2`: 

```
PODS:
  - 1PasswordExtension (1.8.6)
  - Alamofire (4.8.0)
  - Automattic-Tracks-iOS (0.4.4-beta.2): <---
    - CocoaLumberjack (~> 3.5.2)
    - Reachability (~> 3.1)
    - Sentry (~> 4)
    - UIDeviceIdentifier (~> 1)
```

I then checked `TracksConstants.m` and it has this:

```objc
NSString *const TracksLibraryVersion = @"0.4.4-beta.2";
```

I have a feeling I misunderstood the problem. 😄 What am I missing? 🤔

Reply:
🤔 I wonder if the problem is only happening on WPiOS. I'll get back to you on that. I need to ask Jeremy some questions 😄

Reply:
Okay, I don't yet know why, but I can confirm this is only happening on WPiOS. Will remove 🙂